library/database/postgres/cluster.proto

```diff
@@ -0,0 +1,17 @@
+// Copyright 2022 Namespace Labs Inc; All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+
+syntax = "proto3";
+
+package library.database.postgres;
+
+option go_package = "namespacelabs.dev/foundation/library/database/postgres";
+
+message ClusterIntent {
+}
+
+message ClusterInstance {
+    string url      = 1;
+    string password = 2;
```

Comment:
Why TODO? My plan is that this is generated (compare Minio).

Reply:
Because we're not passing a reference to the secret to the application, for it to mount the secret, but rather we're passing it by value. Which means that it will end up injected in the resource configuration. See also NSL-339.